lib/routes/auth/login.js

```diff
@@ -16,21 +15,22 @@ module.exports = {
         },
         handler: async (request, h) => {
 
-            const AuthService = h.services().authService;
+            const { authService, tokenService } = h.services();
             const { username, password } = request.payload;
             const { isDev, jwt: { refreshTokenLifespan } } = Toys.options(request);
 
-            const results = await AuthService.login(username, password);
+            const results = await authService.login(username, password);
 
             if (!results.isBoom) {
                 h.state('_pallies', results.refreshToken, {
                     ttl: refreshTokenLifespan * 1000,
                     isSecure: !isDev,
-                    isHttpOnly: true
+                    isHttpOnly: true,
+                    path: '/login'
```

Comment:
I think this should be
```suggestion
                    path: '/reauthorize'
```